JitsiConferenceEvents.js

```diff
@@ -156,3 +156,7 @@ export const USER_ROLE_CHANGED = "conference.roleChanged";
  * User status changed.
  */
 export const USER_STATUS_CHANGED = "conference.statusChanged";
+/**
+ * Event indicates that local user is talking while he muted himself
+ */
+export const TALK_WHILE_MUTED = "conference.talk_while_muted";
```

Comment:
I prefer alphabetical sorting because it allows easy searching in the fashion of searching through a phone book. Since (1) I've sorted the existing constants in alphabetical order and (2) I see no advantage to adding `TALK_WHILE_MUTED` at the bottom, I'll move it.